File: CS/WinWebSolution.Module.Win/WinSortListViewController.cs
using DevExpress.ExpressApp.Win.Editors;

namespace WinWebSolution.Module.Win {
    public class WinSortListViewController : SortListViewControllerBase {
        protected override void OnViewControlsCreated() {
            base.OnViewControlsCreated();
            GridListEditor gridListEditor = View.Editor as GridListEditor;
            if (gridListEditor != null) {
                gridListEditor.GridView.OptionsCustomization.AllowSort = false;
            }
        }
    }
}

File: VB/WinWebSolution.Module.Win/WinSortListViewController.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.ExpressApp.Win.Editors

Namespace WinWebSolution.Module.Win
	Public Class WinSortListViewController
		Inherits SortListViewControllerBase
		Protected Overrides Sub OnViewControlsCreated()
			MyBase.OnViewControlsCreated()
			Dim gridListEditor As GridListEditor = TryCast(View.Editor, GridListEditor)
			If gridListEditor IsNot Nothing Then
				gridListEditor.GridView.OptionsCustomization.AllowSort = False
			End If
		End Sub
	End Class
End Namespace
